src/patternfly/components/Table/table.scss

```diff
@@ -78,8 +79,6 @@
   --#{$table}--cell--hidden-visible--Display: table-cell;
 
   // * Table toggle
-  --#{$table}__toggle--PaddingBlockStart: var(--pf-t--global--spacer--sm);
```

Comment:
Not sure if you meant to remove this? It's breaking to remove it, and it looks like it's still used here https://github.com/patternfly/patternfly/blob/4ee99f15fbb5760024916e185cda063f19cdda6f/src/patternfly/components/Table/table.scss#L741